tx/textual/valuerenderer/message.go

```diff
@@ -79,6 +98,73 @@ func (mr *messageValueRenderer) Format(ctx context.Context, v protoreflect.Value
 	return screens, nil
 }
 
+func (mr *messageValueRenderer) FormatRepeated(ctx context.Context, vr ValueRenderer, v protoreflect.Value, name string) ([]Screen, error) {
```

Comment:
We could maybe simplify the func signature, and make it private:

```suggestion
func (mr *messageValueRenderer) formatRepeated(ctx context.Context, vr ValueRenderer, v protoreflect.Value, fd protoreflect.FieldDescriptor) ([]Screen, error) {
```

Reply:
Done in dc079c6 ( also removed the `protorelfect.Value` parameter )